### ISSUE TITLE
.arr Packages Restart Problem

### ISSUE THREAD

**Issue by ufuri** [SynoCommunity/spksrc#5370]
I am using Radarr, Prowlarr and Sonarr packages in my DS220+. When I scheculed my device to power off at night and power on in the morning, these packages can not restart automatically. And I have to go package center and "Repair" each of them manually. Then they work flawlessly until next restart. What can cause this?

**Comment by AlbanSeurat** [comment]
I have the same issue. 

Error 1 returned by the service script. (wait_for_status timeout)
log file: /var/log/systemd/pkgctl-radarr.service.log

```
2022-09-02T09:01:20+02:00 XXXXX bash[10993]: ======== start radarr ========
2022-09-02T09:01:42+02:00 XXXXX bash[10993]: Failed to execute '/var/packages/radarr/scripts/start-stop-status start' (err=1)
```


my synology boot time is 09:00
Looking at radarr log  file I see that (trace level setup):
```
2022-09-01 09:01:39.9|Debug|ProcessProvider|Found 0 processes with the name: Radarr.Console
2022-09-01 09:01:40.3|Debug|ProcessProvider|Found 1 processes with the name: Radarr
2022-09-01 09:01:40.3|Debug|ProcessProvider| - [11024] Radarr
2022-09-01 09:01:40.4|Info|MigrationController|*** Migrating data source=/volume1/@appdata/radarr/.config/Radarr/radarr.db;cache size=-20000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
2022-09-02 09:01:41.2|Debug|ProcessProvider|Found 0 processes with the name: Radarr.Console
2022-09-02 09:01:41.4|Debug|ProcessProvider|Found 1 processes with the name: Radarr
2022-09-02 09:01:41.4|Debug|ProcessProvider| - [11580] Radarr
2022-09-02 09:01:41.6|Info|MigrationController|*** Migrating data source=/volume1/@appdata/radarr/.config/Radarr/radarr.db;cache size=-20000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
``` 
And then nothing. 


finally /var/log/synoplugin.log

```
2022-09-02T09:01:20+02:00 XXXXXX synopkgctl[11005]: plugin_action.c:317 synoplugin: [11005][PRE][package_action][MAIN] Scripts=[synomibclient_package_action.sh]; Args=[PKGEVENT={"action":"start","beta":false,"error":{"code":0},"finished":false,"language":"","last_stage":"prepare","package":"radarr","pid":11005,"stage":"prepare_start","status":"starting","success":false,"timestamp":1662102080561,"username":"","version":"20220515-18"},PKGNAME=radarr]
```

2022-09-02T09:01:20+02:00 XXXXX synopkgctl[11218]: plugin_action.c:319 synoplugin: [11005][PRE][package_action][synomibclient_package_action.sh][11218] ExitCode: 0
2022-09-02T09:01:20+02:00 XXXXX synopkgctl[11218]: plugin_action.c:319 synoplugin: [11005][PRE][package_action][synomibclient_package_action.sh][11218] Runtime: 0.203s
2022-09-02T09:01:20+02:00 XXXX synopkgctl[11005]: plugin_action.c:317 synoplugin: [11005][PRE][package_action][MAIN] Runtime: 0.215s
2022-09-02T09:01:21+02:00 XXXX synopkgctl[11005]: plugin_action.c:317 synoplugin: [11005][PRE][package_start][MAIN] Scripts=[remove_jsconfig_cache.sh,synoscheduled_reload.sh]; Args=[PKGVERSION=20220515-18,PKGNAME=radarr,PKGACTION=none]
2022-09-02T09:01:21+02:00 XXXX synopkgctl[11421]: plugin_action.c:319 synoplugin: [11005][PRE][package_start][remove_jsconfig_cache.sh][11421] ExitCode: 0
2022-09-02T09:01:21+02:00 XXXX synopkgctl[11421]: plugin_action.c:319 synoplugin: [11005][PRE][package_start][remove_jsconfig_cache.sh][11421] Runtime: 0.030s
2022-09-02T09:01:21+02:00 XXXX synopkgctl[11422]: plugin_action.c:319 synoplugin: [11005][PRE][package_start][synoscheduled_reload.sh][11422] ExitCode: 0
2022-09-02T09:01:21+02:00 XXXX synopkgctl[11422]: plugin_action.c:319 synoplugin: [11005][PRE][package_start][synoscheduled_reload.sh][11422] Runtime: 0.030s
2022-09-02T09:01:21+02:00 XXXX synopkgctl[11005]: plugin_action.c:317 synoplugin: [11005][PRE][package_start][MAIN] Runtime: 0.074s
2022-09-02T09:01:42+02:00 XXXX synopkgctl[11005]: plugin_action.c:317 synoplugin: [11005][POST][package_action][MAIN] Scripts=[synomibclient_package_action.sh]; Args=[PKGEVENT={"action":"start","available_operation":{},"beta":false,"error":{"code":272,"description":"Failed to run script, script=[start]"},"finished":true,"language":"","last_stage":"prepare_start","package":"radarr","pid":11005,"scripts":[{"code":1,"message":"","type":"start"}],"stage":"start_failed","status":"start_failed","status_code":272,"status_description":"failed to start on previous startup","success":false,"timestamp":1662102102705,"username":"","version":"20220515-18"},PKGNAME=radarr]
2022-09-02T09:01:42+02:00 XXXX synopkgctl[14003]: plugin_action.c:319 synoplugin: [11005][POST][package_action][synomibclient_package_action.sh][14003] ExitCode: 0
2022-09-02T09:01:42+02:00 XXXX synopkgctl[14003]: plugin_action.c:319 synoplugin: [11005][POST][package_action][synomibclient_package_action.sh][14003] Runtime: 0.015s
2022-09-02T09:01:42+02:00 XXXX synopkgctl[11005]: plugin_action.c:317 synoplugin: [11005][POST][package_action][MAIN] Runtime: 0.018s

**Comment by ufuri** [comment]
I've solved it temporarily by starting every .arr package service with schedule it after 5 minutes later of power on NAS.

**Comment by mreid-tt** [comment]
I also have a similar ticket for this. In my checks I've found that I was able to fix Sonarr requiring a repair by uninstalling it and reinstalling it clean. This is a bit tricky since Sonarr does not seem to clean out its `appdata` as it should. I've outlined the specific steps I did in this post: https://github.com/SynoCommunity/spksrc/issues/5133#issuecomment-1274710664.

Let me know if that works for you...
